src/commonmark/en/content/user/importexport-app.md

```diff
@@ -27,8 +27,12 @@ which can be found below.
 
 ## Importing data
 
+<!--DHIS2-SECTION-ID:importing_data-->
+
 ### Import progress logger
 
+<!--DHIS2-SECTION-ID:import_process_logger-->
```

Comment:
progress or process?
(doesn't really matter, but nice to be consistent).

Reply:
Good catch

Reply:
Fixed